From f9d31febf87e2ae0050503f302527419a15cb29c Mon Sep 17 00:00:00 2001
From: Weng Xuetian <wengxt@gmail.com>
Date: Mon, 20 Apr 2020 13:54:26 -0700
Subject: Update the visiblity check for ActionToolbar by checking width with
 less-"equal".

Summary:
Otherwise some action, especially when there is only one action, it will be
put in "more" menu even if there is enough space.

Reviewers: #kirigami, mart, ngraham

Reviewed By: #kirigami, ngraham

Subscribers: ngraham, plasma-devel

Tags: #kirigami

Differential Revision: https://phabricator.kde.org/D29022
---
 src/controls/private/ActionToolBarLayoutDetails.qml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/controls/private/ActionToolBarLayoutDetails.qml b/src/controls/private/ActionToolBarLayoutDetails.qml
index 53b76a1..e213c6e 100644
--- a/src/controls/private/ActionToolBarLayoutDetails.qml
+++ b/src/controls/private/ActionToolBarLayoutDetails.qml
@@ -118,7 +118,7 @@ Item {
                     return true
                 }
 
-                property bool actionVisible: visible && (x + width < details.fullLayoutWidth)
+                property bool actionVisible: visible && (x + width <= details.fullLayoutWidth)
 
                 onLoaded: {
                     if (sourceComponent == toolButtonDelegate) {
-- 
cgit v1.1

